storage/replica.go

```diff
@@ -193,11 +208,13 @@ var _ client.Sender = &Replica{}
 // NewReplica initializes the replica using the given metadata.
 func NewReplica(desc *roachpb.RangeDescriptor, rm *Store) (*Replica, error) {
 	r := &Replica{
-		store:       rm,
-		cmdQ:        NewCommandQueue(),
-		tsCache:     NewTimestampCache(rm.Clock()),
-		sequence:    NewSequenceCache(desc.RangeID),
-		pendingCmds: map[cmdIDKey]*pendingCmd{},
+		store:          rm,
+		cmdQ:           NewCommandQueue(),
+		tsCache:        NewTimestampCache(rm.Clock()),
+		sequence:       NewSequenceCache(desc.RangeID),
+		pendingCmds:    map[cmdIDKey]*pendingCmd{},
+		applyQueueChan: make(chan committedCmd, applyQueueChanSize),
+		stopFlag:       true,
```

Comment:
the default value for `stopFlag` can go with my suggestion (since the zero value is the default).